etna/analysis/plotters.py

```diff
@@ -1132,6 +1159,8 @@ def plot_feature_relevance(
         :py:class:`~etna.analysis.feature_relevance.relevance.RelevanceTable`
     top_k:
         number of best features to plot, if None plot all the features
+    alpha:
+        significance level, default alpha = 0.05, only for StatisticsRelevanceTable
```

Comment:
not StatisticsRelevanceTable but :py:class:`~etna.analysis.feature_relevance.relevance.StatisticsRelevanceTable` it will render nicely in the code